.changeset/serious-kings-decide.md

```diff
@@ -0,0 +1,9 @@
+---
+"@salt-ds/lab": minor
+---
+
+enabled uncontrolled/un-controlled open behaviour for `DatePicker`
+
+- added `openOnClick`, `openOnKeyDown` and `openOnFocus` props to `DatePicker`.
```

Comment:
I'm worried adding all of these initially might be a bad idea, `openOnFocus` is the one that worries me the most as I think it's the one that most teams will use incorrectly and the UX for it isn't great.

Reply:
openOnFocus is needed for migration as that was provided by UITK

Reply:
removed for the mo... if folks needs this, they can achieve through, `interactions` props